index.rst

```diff
@@ -3,16 +3,26 @@ BookServer documentation
 ************************
 :index:`docs to write`: The BookServer is a next-generation server for the Runestone platform...
 
-
-Core web application
-====================
 See the `genindex` for todo, fixme, etc. items.
 
+
+Documentation
+=============
 .. toctree::
     :maxdepth: 1
 
     docs/installation
     docs/goals
+    docs/design
```

Comment:
the descriptions of the files I have in here may be better to add to the toctree in the app folder??  That would keep it from being disjointed.

I would like this to be readable and useful.  So maybe spending some time up front thinking about where we put detail would be useful.

Reply:
I went back and forth on this -- at first, the were part of the toctree. Then, at three files, it seems better to have them under a separate heading. However, if we combine goals and design, I can see putting these back together.

I'll put them back in the same heading as the code for now, and we can always move them based on what seems to provide a good flow.